shell/platform/darwin/ios/framework/Source/FlutterStandardCodec.mm

```diff
@@ -36,34 +55,53 @@ - (id)decode:(NSData*)message {
 
 #pragma mark - Codec for method channel
 
-@implementation FlutterStandardMethodCodec
+@implementation FlutterStandardMethodCodec {
+  FlutterStandardReaderWriter* _readerWriter;
+}
 + (instancetype)sharedInstance {
   static id _sharedInstance = nil;
   if (!_sharedInstance) {
-    _sharedInstance = [FlutterStandardMethodCodec new];
+    _sharedInstance =
+        [FlutterStandardMethodCodec withReaderWriter:[FlutterStandardReaderWriter new]];
   }
   return _sharedInstance;
 }
 
++ (instancetype)withReaderWriter:(FlutterStandardReaderWriter*)readerWriter {
```

Comment:
It is not idiomatic Objective-C to begin a selector name with a preposition. `codecWithReaderWriter:` or even `standardMethodCodecWithReaderWriter:` sounds more idiomatic.